From: Tian Tao <tiantao6@hisilicon.com>
Date: Thu, 24 Sep 2020 10:20:18 +0800
Subject: efi: Delete deprecated parameter comments
Patch-mainline: v5.10-rc1
Git-commit: f5344e5d6ccb9ddf377202690a135bc64607c621
References: jsc#SLE-16407

Delete deprecated parameter comments to  fix warnings reported by make
W=1.
drivers/firmware/efi/vars.c:428: warning: Excess function parameter
'atomic' description in 'efivar_init'

Signed-off-by: Tian Tao <tiantao6@hisilicon.com>
Link: https://lore.kernel.org/r/1600914018-12697-1-git-send-email-tiantao6@hisilicon.com
Signed-off-by: Ard Biesheuvel <ardb@kernel.org>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/firmware/efi/vars.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/firmware/efi/vars.c
+++ b/drivers/firmware/efi/vars.c
@@ -403,7 +403,6 @@ static void dup_variable_bug(efi_char16_
  * efivar_init - build the initial list of EFI variables
  * @func: callback function to invoke for every variable
  * @data: function-specific data to pass to @func
- * @atomic: do we need to execute the @func-loop atomically?
  * @duplicates: error if we encounter duplicates on @head?
  * @head: initialised head of variable list
  *
